ts/screens/modal/RootModal.tsx

```diff
@@ -28,9 +33,54 @@ type Props = ReturnType<typeof mapStateToProps>;
 const RootModal: React.FunctionComponent<Props> = (props: Props) => {
   const dispatch = useDispatch();
 
+  if (!props.isDeviceSupported) {
+    return <UnsupportedDeviceScreen />;
+  }
+  // avoid app usage if backend systems are OFF
+  if (props.isBackendServicesStatusOff) {
+    return <SystemOffModal />;
+  }
+  // if the app is out of date, force a screen to update it
+  if (!props.isAppSupported) {
+    void mixpanelTrack("UPDATE_APP_MODAL", {
+      minVersioniOS: props.versionInfo?.min_app_version.ios,
+      minVersionAndroid: props.versionInfo?.min_app_version.android
+    });
+    return <UpdateAppModal />;
+  }
+
+  if (props.tokenRefreshing.kind === "transient-error") {
+    return (
+      <AskUserInteractionScreen
+        pictogramName="umbrella"
+        title={I18n.t(
+          "fastLogin.userInteraction.sessionExpired.transientError.title"
+        )}
+        subtitle={I18n.t(
+          "fastLogin.userInteraction.sessionExpired.transientError.subtitle"
+        )}
+        onSubmit={() => {
+          openWebUrl("https://io.italia.it/faq");
```

Comment:
We should trace the implementation of this url with a Jira ticket and handle it via env file when we have the official one available. What do you think?

Reply:
Yup! You are right! 🙏

Reply:
Fixed in 7a75bf1851188de4c02af169539429e89e4ad75d